client/src/components/mixins/nr-affiliation-mixin.ts

```diff
@@ -0,0 +1,63 @@
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import AuthServices from '@/services/auth.services'
+import BusinessServices from '@/services/business.services'
+import newReqModule from '@/store/new-request-module'
+import { BusinessRequest, CreateNRAffiliationRequestBody } from '@/models/business'
+import { NameRequestI } from '@/models'
+
+@Component
+export default class NrAffiliationMixin extends Vue {
+  /**
+   * Create an affiliation between current authenticated Account and current Nr
+   * @param nr The Nr to affiliate
+   */
+  async createAffiliation (nr: NameRequestI): Promise<any> {
+    try {
+      const currentOrganizationId = JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT')).id
+      const requestBody: CreateNRAffiliationRequestBody = {
+        businessIdentifier: nr.nrNum,
+        phone: nr.applicants?.phoneNumber || '',
+        email: nr.applicants?.emailAddress || ''
+      }
+
+      // Request to affiliate NR to current account
+      const nrResponse = await AuthServices.createNRAffiliation(currentOrganizationId, requestBody)
+      if (nrResponse?.status === 201) {
+      // update the legal api if the status is success
```

Comment:
Possible lint error (indentation). Run `npm run lint` before your final commit.

Reply:
Linter didn't pick it up, but i will fix/indent here, thanks!